examples/About.ipynb

```diff
@@ -8,18 +8,39 @@
     "[holoviz.org](https://holoviz.org) website shows how to use Param\n",
     "together with other libraries to solve complex problems, with detailed\n",
     "tutorials and examples. Example projects using Param are at\n",
-    "[examples.pyviz.org](https://examples.pyviz.org).\n",
+    "[examples.pyviz.org](https://examples.pyviz.org). and you can compare Param to other available tools at [pyviz.org](https://pyviz.org).\n",
```

Comment:
Here there is probably no reason to link to pyviz.org, because param isn't a viz library, and doesn't appear there...

Reply:
I removed it.